lib/Controller/OutboxController.php

```diff
@@ -180,6 +188,9 @@ public function update(int     $id,
 		$message->setFailed($failed);
 		$message->setInReplyToMessageId($inReplyToMessageId);
 		$message->setSendAt($sendAt);
+		$message->setSmimeSign($smimeSign);
+		$message->setSmimeCertificateId($smimeCertificateId);
```

Comment:
should you check if it belongs to the user?

Reply:
Good catch, better safe than sorry.